src/components/ShareBlock/ShareBlock.tsx

```diff
@@ -85,80 +83,64 @@ const ShareBlock = ({
-                      size={40}
-                      round={false}
-                      bgStyle={{ fill: 'auto' }}
-                    />
-                  </LinkedinShareButton>
-                </StyledShareButton>
-              </ShareButtonContainer>
-              <EmbedPrompt>
-                Or <span onClick={onClickEmbed}>embed on your website</span>
-              </EmbedPrompt>
-            </SocialTextArea>
-          </SocialTextAreaWrapper>
-          <SocialMockupWrapper>{SocialPreview}</SocialMockupWrapper>
-        </ShareRowContentArea>
-      </ShareRow>
+      {!region && (
+        <ShareRow newsletter={false}>
+          <ShareRowContentArea
+            $isMatchingProjectionsRoute={isMatchingProjectionsRoute !== null}
+          >
+            <SocialTextAreaWrapper id="shareblock">
+              <SocialTextArea>
+                <ShareInstructionHeader>
+                  Share COVID risk in your community
+                </ShareInstructionHeader>
+                <ShareInstructionBody>
+                  Share real-time, local COVID data with your neighbors,
+                  friends, and family.
+                </ShareInstructionBody>
+                <ShareButtonContainer reflow>
+                  <StyledShareButton variant="contained" color="#3b5998">
+                    <FacebookShareButton
+                      url={url}
+                      quote={quote}
+                      beforeOnClick={() => {
+                        trackShare('facebook');
+                        return Promise.resolve();
+                      }}
+                    >
+                      <FacebookIcon
+                        size={40}
+                        round={false}
+                        bgStyle={{ fill: 'auto' }}
+                      />
+                    </FacebookShareButton>
+                  </StyledShareButton>
+                  <StyledShareButton variant="contained" color="#00acee">
+                    <TwitterShareButton
+                      url={url}
+                      title={quote}
+                      hashtags={[hashtag]}
+                      beforeOnClick={() => {
+                        trackShare('twitter');
+                        return Promise.resolve();
+                      }}
+                    >
+                      <TwitterIcon
+                        size={40}
+                        round={false}
+                        bgStyle={{ fill: 'auto' }}
+                      />
+                    </TwitterShareButton>
+                  </StyledShareButton>
+                </ShareButtonContainer>
+                <EmbedPrompt>
+                  Or <span onClick={onClickEmbed}>embed on your website</span>
+                </EmbedPrompt>
+              </SocialTextArea>
+            </SocialTextAreaWrapper>
+            <SocialMockupWrapper>{SocialPreview}</SocialMockupWrapper>
+          </ShareRowContentArea>
+        </ShareRow>
+      )}
```

Comment:
We could move the `<ShareRow>` component to its own file and conditionally include it in `ShareModelBlock` instead, since that part doesn't really need the `region`. Feel free to punt for later!